### DISCUSSION TITLE
Why 512 ms delay at AriaAnnouncer.vue ?

### DISCUSSION THREAD

**Discussion by yamanoku** [elk-zone/elk#2402] [Q&A]
Hi, I was code-reading when I heard that elk is an accessible application.
I like the way [AriaAnnouncer](https://github.com/elk-zone/elk/blob/b723d51786d7767510dd3eff62b72fc339d0c2cb/components/aria/AriaAnnouncer.vue) is made! I think it is thoughtful to screen reader users.

I was looking at the actual code, why are you delaying 512 ms when the title reads?

https://github.com/elk-zone/elk/blob/b723d51786d7767510dd3eff62b72fc339d0c2cb/components/aria/AriaAnnouncer.vue#L46

**Comment by userquin** [comment]
we should use debounce + title watcher instead setTimeout, it was the initial implementation

The title can change in a few places in some pages, since we're composing them: check for example settings pages, there are a few useHead usages (IIRC).
